ethereumjs/ethereumjs-monorepo
# `CLRequests` cleanup

Clean-up following integration of EIP 6110, 7002, and 7685 code
 
- Moves all `CLRequest` related code to `vm/src/requests.ts`
- Adds detailed explanation of parsing Deposit event logs
- Cleans up field/function naming
- Adds prague config to test-runner from #3387 
- Add better forkTime check for EIP 2935
- Make test runner use Typescript code instead of `dist/cjs/...`
- Add optional `depositContractAddress` field to `ChainConfig` in `common`
